Formula/libdatrie.rb

```diff
@@ -0,0 +1,35 @@
+class Libdatrie < Formula
+  desc "Double-Array Trie Library"
+  homepage "https://linux.thai.net/~thep/datrie/datrie.html"
+  url "https://github.com/tlwg/libdatrie/releases/download/v0.2.13/libdatrie-0.2.13.tar.xz"
+  sha256 "12231bb2be2581a7f0fb9904092d24b0ed2a271a16835071ed97bed65267f4be"
+  license "LGPL-2.1-or-later"
+
+  def install
+    # 'make install' fails in parallel mode
+    ENV.deparallelize
+
+    # upstream ./configure does not support --disable-debug
+    system "./configure", *std_configure_args.delete("--disable-debug"), "--disable-silent-rules"
```

Comment:
Oops, this doesn't work. Sorry for the bad suggestion!

Reply:
Ah, will fix 👍

Reply:
10136f0befb